src/modules/modals/EditContext.vue

```diff
@@ -209,6 +223,37 @@ export default {
 			}
 			return resources
 		},
+		async actionDeleteContext() {
+			this.prepareDeleteContext = false
+			const res = await this.$store.dispatch('removeContext', { context: this.context, receivers: this.context.sharing })
```

Comment:
This throws an error as this.context doesn't seem to be defined in here. The following seems to work:

```
--- a/src/modules/modals/EditContext.vue
+++ b/src/modules/modals/EditContext.vue
@@ -225,7 +225,8 @@ export default {
                },
                async actionDeleteContext() {
                        this.prepareDeleteContext = false
-                       const res = await this.$store.dispatch('removeContext', { context: this.context, receivers: this.context.sharing })
+                       const context = this.getContext(this.contextId)
+                       const res = await this.$store.dispatch('removeContext', { context, receivers: context.sharing })
                        if (res) {
                                showSuccess(t('tables', 'Application "{context}" removed.', { context: this.title }))
                                // if the active context was deleted, go to startpage
```

Reply:
good catch, thx